Fortigate IP: 100.64.14.11
Response data:
Test 1 URL: https://100.64.14.11:10443/api/v2/monitor/system/status

Test 1 Error: HTTPSConnectionPool(host='100.64.14.11', port=10443): Max retries exceeded with url: /api/v2/monitor/system/status (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x0000026098F3AED0>, 'Connection to 100.64.14.11 timed out. (connect timeout=None)'))
Response Data: API call failed with the keyFortigate IP: 100.64.14.11
No username provided, cannot log in.
